systems/plants/@RigidBodyManipulator/getCMMdA.m

```diff
@@ -1,8 +1,14 @@
-function [A,dA] = getCMMdA(model, kinsol, robotnum, in_terms_of_qdot)
-% returns the centroidal momentum matrix as described in Orin & Goswami 2008
+function [A, dA] = getCMMdA(robot, kinsol, robotnum, in_terms_of_qdot)
```

Comment:
I like the longer name.  Please do keep the getCMM stub and throw the deprecated.  I don't think we need to do that for HandC, etc, since they were not part of the public interface for RBM in matlab.  But getCMM was.



> On Apr 16, 2015, at 10:45 AM, Twan Koolen <notifications@github.com> wrote:
> 
> In systems/plants/@RigidBodyManipulator/getCMMdA.m:
> 
> > @@ -1,8 +1,14 @@
> > -function [A,dA] = getCMMdA(model, kinsol, robotnum, in_terms_of_qdot)
> > -% returns the centroidal momentum matrix as described in Orin & Goswami 2008
> > +function [A, dA] = getCMMdA(robot, kinsol, robotnum, in_terms_of_qdot)
> Actually, I'm in favor of centroidalMomentumMatrix as the method name. I called it that in C++. It has no abbreviations. It doesn't have 'get' in it, which implies to me that it's already precomputed and hence comes at no computational cost.
> getCMMdA never had qd as an input (not necessary since the centroidal momentum matrix and its gradient aren't velocity dependent). I got rid of all getCMM calls in Drake and I got rid of getCMM, so calling that method (which did have qd as an input) simply results in an error right now. Is that good enough or do you want to keep a getCMM method and have it throw a 'deprecated' error with instructions on how to change your code?
> 
> —
> Reply to this email directly or view it on GitHub.
>